### DISCUSSION TITLE
Latency between scheduling and starting orchestrator

### DISCUSSION THREAD

**Discussion by sibircevs** [Azure/azure-functions-durable-extension#2403] [Q&A]
Hi,

I saw this 4 year old issue.
https://github.com/Azure/azure-functions-durable-extension/issues/449

but now I have the same problem and it happens not once.
![image](https://user-images.githubusercontent.com/25064299/221605351-3315d021-70ae-4fb1-b10a-d8e5483cdc44.png)

It is huge difference between schedule and start time.
What should I check or what kind of problem can it be?

BR,
Andrejs

**Comment by cgillum** [comment]
Basic information for possible causes of orchestration start delays can be found here: https://learn.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-azure-storage-provider#orchestration-start-delays. However, your case is likely different because your delay is quite a bit longer.

Since the delay is almost exactly 5 minutes and 30 seconds, my guess is that there was some internal failure that caused the message to get dequeued, not processed, and then picked up again after the 5-minute queue visibility delay expired.

I recommend enabling Durable Task Framework logs to see if you can find any warnings related to this instance ID. More info here: https://learn.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-diagnostics?tabs=csharp-inproc#durable-task-framework-logging

1. In the storage account you configured for Durable Functions, is there a large backlo
/cc @lilyjma as this will be a good scenario to include in the troubleshooting guide.

**Reply by sibircevs** [reply]
@cgillum 
Hi,
I want to return to this problem.
I frequently have such problems. There is an example of the last one:
![image](https://user-images.githubusercontent.com/25064299/234032363-ba8c4693-5991-47ac-ac7e-f48cf45889fd.png)

**Reply by cgillum** [reply]
Without further context, I think my previous response is still the best I can offer right now. If you want help debugging this, I suggest opening a GitHub issue or a support ticket with Azure.
